pom.xml

```diff
@@ -309,7 +309,6 @@
         <property>
           <name>qa</name>
         </property>
-        <jdk>(1.8,)</jdk>
```

Comment:
We must support both jdk 8 and jdk11 builds with qa enabled.

Reply:
Yes. 
Maven isn't my primary build system so I don't have under the hood knowledge on its inner workings. 
Please explain to the funcional behaviour.

From my prevous testing by trail and error approach it seemed ok.
Here's what i get:
```
C:\dev\geotools-kartverket>mvn --projects org.geotools.ogc:org.w3.xlink help:active-profiles
[INFO] Scanning for projects...
[WARNING]
[WARNING] Some problems were encountered while building the effective model for org.geotools:gt-coverage-api:jar:22-SNAPSHOT
[WARNING] 'dependencies.dependency.version' for com.github.spotbugs:spotbugs-annotations:jar is either LATEST or RELEASE (both of them are being deprecated) @ org.geotools:gt-coverage-api:[unknown-version], C:\dev\geotools-kartverket\modules\plugin\coverage-multidim\coverage-api\pom.xml, line 124, column 20
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
[INFO]
[INFO] -------------------< org.geotools.ogc:org.w3.xlink >--------------------
[INFO] Building Xlink Model 22-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-help-plugin:3.1.1:active-profiles (default-cli) @ org.w3.xlink ---
[INFO]
Active Profiles for Project 'org.geotools.ogc:org.w3.xlink:jar:22-SNAPSHOT':

The following profiles are active:

 - errorprone-jdk8-fix (source: org.geotools:geotools:22-SNAPSHOT)

[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
```


```
C:\dev\geotools-kartverket>mvn --projects org.geotools.ogc:org.w3.xlink help:active-profiles -Dqa
[INFO] Scanning for projects...
[WARNING]
[WARNING] Some problems were encountered while building the effective model for org.geotools:gt-coverage-api:jar:22-SNAPSHOT
[WARNING] 'dependencies.dependency.version' for com.github.spotbugs:spotbugs-annotations:jar is either LATEST or RELEASE (both of them are being deprecated) @ org.geotools:gt-coverage-api:[unknown-version], C:\dev\geotools-kartverket\modules\plugin\coverage-multidim\coverage-api\pom.xml, line 124, column 20
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
[INFO]
[INFO] -------------------< org.geotools.ogc:org.w3.xlink >--------------------
[INFO] Building Xlink Model 22-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-help-plugin:3.1.1:active-profiles (default-cli) @ org.w3.xlink ---
[INFO]
Active Profiles for Project 'org.geotools.ogc:org.w3.xlink:jar:22-SNAPSHOT':

The following profiles are active:

 - pmd (source: org.geotools:geotools:22-SNAPSHOT)
 - errorprone (source: org.geotools:geotools:22-SNAPSHOT)
 - errorprone-jdk8-fix (source: org.geotools:geotools:22-SNAPSHOT)
 - spotbugs (source: org.geotools:geotools:22-SNAPSHOT)
 - checkstyle (source: org.geotools:geotools:22-SNAPSHOT)


[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------

```


```
C:\dev\geotools-kartverket>mvn --projects org.geotools.ogc:org.w3.xlink help:active-profiles -P errorprone
[INFO] Scanning for projects...
[WARNING]
[WARNING] Some problems were encountered while building the effective model for org.geotools:gt-coverage-api:jar:22-SNAPSHOT
[WARNING] 'dependencies.dependency.version' for com.github.spotbugs:spotbugs-annotations:jar is either LATEST or RELEASE (both of them are being deprecated) @ org.geotools:gt-coverage-api:[unknown-version], C:\dev\geotools-kartverket\modules\plugin\coverage-multidim\coverage-api\pom.xml, line 124, column 20
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
[INFO]
[INFO] -------------------< org.geotools.ogc:org.w3.xlink >--------------------
[INFO] Building Xlink Model 22-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-help-plugin:3.1.1:active-profiles (default-cli) @ org.w3.xlink ---
[INFO]
Active Profiles for Project 'org.geotools.ogc:org.w3.xlink:jar:22-SNAPSHOT':

The following profiles are active:

 - errorprone (source: org.geotools:geotools:22-SNAPSHOT)
 - errorprone-jdk8-fix (source: org.geotools:geotools:22-SNAPSHOT)



[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------

```